grpc-sys/src/grpc_wrap.rs

```diff
@@ -93,28 +93,31 @@ impl grpc_byte_buffer_reader {
     }
 }
 
-impl From<i32> for grpc_status_code {
-    fn from(code: i32) -> grpc_status_code {
+
+pub struct GrpcStatusCode(u64);
```

Comment:
How about `GrpcStatusCode(pub u64)`?

Reply:
> Please don't use Chinese in your comments

Forget it, just a test...

Reply:
Yes but that's gonna get stored in GitHub jquery cache for some sense (in case you can delete them)

Reply:
Covered by `--force`

Reply:
:+1: